modules/life-cycle-limited-support.adoc

```diff
@@ -5,11 +5,18 @@
 [id="rosa-limited-support_{context}"]
 = Limited support status
 
-While operating outside of the supported versions list, you may be asked to upgrade the cluster to
-a supported version when requesting support. Additionally, Red Hat does not make any runtime or SLA
-guarantees for clusters outside of the supported versions list at the end of their 9 month end of
-life cycle date.
-
-Red Hat provides commercially reasonable support to ensure an upgrade path from an unsupported
-release to a supported release is available. However, if a supported upgrade path is no longer
-available, you may be required to create a new cluster and migrate your workloads.
+When a cluster transitions to a _Limited Support_ status, it means that the SLA is no longer applicable and credits requested against the SLA are denied. It does not mean that you no longer have product support. The cluster can return to a fully-supported status if the violating factors are remediated.
+
+A cluster transitions to a _Limited Support_ status in the following scenarios:
+
+* If you do not upgrade a cluster to a supported version before the end-of-life date
++
+Red Hat does not make any runtime or SLA guarantees for versions after their end-of-life date. To receive continued support, upgrade the cluster to a supported version within the 9 month period. If you do not upgrade the cluster within the 9 month period, the cluster transitions to a _Limited Support_ status until you upgrade it to a supported version.
++
+Red Hat provides commercially reasonable support to upgrade from an unsupported version to a supported version. However, if a supported upgrade path is no longer available, you might have to create a new cluster and migrate your workloads.
+
+* If you remove or replace any native {product-title} components or any other component that is installed and managed by Red Hat
```

Comment:
ISG says to avoid the use of the word "native" unless the term is part of the existing product terminology, for example, “cloud native”. The suggested words they have for alternatives ("local", "basic", "original", or "default") don't make sense here (imo), but the way this is written, it sounds like native components are a subset of components that are installed and managed by Red Hat. Therefore, would this work?
```suggestion
* If you remove or replace any components that are installed and managed by Red Hat
```

Reply:
Retained the existing term, to specifically call out the native ROSA components as a special subset of the components RH manages per discussion with Jeana. Thank you.